### ISSUE TITLE
Absolute value notation

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/number-line-integers#2]
The mockups use absolute value notation from an equation editor, as opposed to the pipe symbol on the keyboard (ex: |-3| = 3). I'd like to try to match the look of the equation editor, notably the height of the absolute value lines. 
![image](https://user-images.githubusercontent.com/7852916/57542807-ea0dfe00-730f-11e9-930e-52f6a8ea6375.png)

The pipe symbol isn't quite tall enough.

**Comment by jbphet** [comment]
I've modified the way the equation is created to that it uses a Scenery `Line` node for the absolute value indicators.  This line node is made to match the bounds of the number characters, which protrude a bit above and below what is visible.  This seems to work pretty well.  Here are some screenshots for comparison.

Before:

![image](https://user-images.githubusercontent.com/3082831/66685655-651a6e00-ec3a-11e9-862f-a5649046eba7.png)

After (the white background was also added since the last dev version):

![image](https://user-images.githubusercontent.com/3082831/66685702-7a8f9800-ec3a-11e9-8d61-7ffcf595b87e.png)
